src/sass/_patterns-notification.scss

```diff
@@ -25,9 +25,11 @@
         background-color: $color-transparent;
         background-image: url('#{$asset-path}66542f40-close.svg');
         background-size: 1rem;
+        background-repeat: no-repeat;
         border: 0;
         cursor: pointer;
         height: 1rem;
+        padding: 0;
```

Comment:
So, I have no issue with the code or anything but removing the `padding` really bothers me. If you are to try and click this on a smaller device it's not accessible, you'll need precise accuracy. Also there isn't a decent enough hover state.

Being _finger friendly_ should be in our thoughts here. The average touch pixel sizes which are comfortable are around 45 to 57 pixels. We should be following this for better accessibility and UX. Currently this close icon would be set at 16 pixels.

Here is an MIT study regarding touch http://touchlab.mit.edu/publications/2003_009.pdf